README.md

```diff
@@ -1,24 +1,119 @@
-# README
+## WanderPal
+> WanderPal is a RESTful API that allows users to reserve a trip.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## API Manipulation and Endpoints
 
-Things you may want to cover:
+If you need further information about the endpoints you can view the documentation navigating to /api-docs.
+If you would like to clone and view it on your side then try `http://localhost:3000/api-docs`.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Ruby on Rails
+- PostgreSQL
 
-* Configuration
+## Frontend Repository Link
+<a href="https://github.com/danielamoreno699/wanderpal-frontend">WanderPal Frontend</a>
 
-* Database creation
+</div>
 
-* Database initialization
+## Kanban Board
 
-* How to run the test suite
+### Current state
 
-* Services (job queues, cache servers, search engines, etc.)
+[Link to the Kanban Board](https://github.com/users/danielamoreno699/projects/1)
 
-* Deployment instructions
+### Initial State
 
-* ...
+![image](./app/assets/initial%20state.png)
+
+<br>
+<br>
+
+- This final group capstone was completed by a team of 4 members <br>
+        1.       Daniela Moreno <br>
+        2.       Salman Ahmad Khan <br>
+        3.       Felipe Haybar Valenzuela <br>
+        4.       Jules Edozie <br>
+        
+ <br>
+
+## Getting Started
+
+- [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/) on your local machine 
+- [Install Ruby on Rails](https://guides.rubyonrails.org/v5.1/getting_started.html)
+- Clone [this repository](https://github.com/danielamoreno699/WanderPal)
+  ```
+  git clone git@github.com:danielamoreno699/WanderPal.git
+  ```
+- To get started, In the root directory run:
+  ```
+  gem install bundler
+  ```
+  Then run:
+  ```
+  bundle install
+  ```
+  To start the server run: 
+
+  ```
+  rails s
+  ```
+  Open the browser with `localhost:3000`
+
+  
+## After installing bundle
+
+Please run the following commands `rails db:create` `rails db:migrate` `rails s`
+
+## Tests
+```
+bundle exec rspec
+```
+Or
+```
+rspec spec
```

Comment:
- [ ] When I setup the project  and run the tests according to your instructions, this happens:

![image](https://github.com/danielamoreno699/WanderPal/assets/57421823/99d9ebee-b573-4fe0-8979-328234d3336e)

![image](https://github.com/danielamoreno699/WanderPal/assets/57421823/476a2728-1a09-41f5-84a1-f7a147d97942)

Kindly, make it so your all backend tests pass in order to ensure your app's integrity. 👍